irc/znc.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2019 Shivaram Lingamneni <slingamn@cs.stanford.edu>
+// released under the MIT license
+
+package irc
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type zncCommandHandler func(client *Client, command string, params []string, rb *ResponseBuffer)
+
+var zncHandlers = map[string]zncCommandHandler{
+	"*playback": zncPlaybackHandler,
+}
+
+func zncPrivmsgHandler(client *Client, command string, privmsg string, rb *ResponseBuffer) {
+	zncModuleHandler(client, command, strings.Fields(privmsg), rb)
+}
+
+func zncModuleHandler(client *Client, command string, params []string, rb *ResponseBuffer) {
+	command = strings.ToLower(command)
+	if subHandler, ok := zncHandlers[command]; ok {
+		subHandler(client, command, params, rb)
+	} else {
+		rb.Add(nil, "*status!znc@znc.in", "NOTICE", rb.target.Nick(), fmt.Sprintf(client.t("No such module [%s]"), command))
```

Comment:
I wonder whether we should add a bit of text to this like: `"- Oragono does not duplicate this service"` just to make sure real users don't get too confused at the behaviour. Even though we should only be responding to `/ZNC` calls with this, may be a good distinction to make.

Reply:
Good point.

I'm thinking of sending the current line as-is (in case clients are parsing it), plus an additional notice from the server that says "Oragono does not emulate the ZNC module %s". Thoughts?

Reply:
Yeah that sounds like a good way to handle it, for sure.